src/machine/board_badger2040.go

```diff
@@ -0,0 +1,81 @@
+//go:build badger2040
+// +build badger2040
+
+// This contains the pin mappings for the Badger 2040 Connect board.
+//
+// For more information, see: https://shop.pimoroni.com/products/badger-2040
+// Also
+// - Badger 2040 schematic: https://cdn.shopify.com/s/files/1/0174/1800/files/badger_2040_schematic.pdf?v=1645702148
+//
+package machine
+
+const (
+	LED Pin = GPIO25
+
+	BUTTON_A    Pin = GPIO12
+	BUTTON_B    Pin = GPIO13
+	BUTTON_C    Pin = GPIO14
+	BUTTON_UP   Pin = GPIO15
+	BUTTON_DOWN Pin = GPIO11
+	BUTTON_USER Pin = GPIO23
+
+	EPD_BUSY_PIN  Pin = GPIO26
+	EPD_RESET_PIN Pin = GPIO21
+	EPD_DC_PIN    Pin = GPIO20
+	EPD_CS_PIN    Pin = GPIO17
+	EPD_SCK_PIN   Pin = GPIO18
+	EPD_SDO_PIN   Pin = GPIO19
+
+	VBUS_DETECT Pin = GPIO24
+	BATTERY     Pin = GPIO29
+	ENABLE_3V3  Pin = GPIO10
+)
+
+// I2C pins
+const (
+	I2C0_SDA_PIN Pin = GPIO4
+	I2C0_SCL_PIN Pin = GPIO5
+
+	I2C1_SDA_PIN Pin = NoPin
+	I2C1_SCL_PIN Pin = NoPin
+)
+
+// SPI pins.
+const (
+	SPI0_SCK_PIN Pin = NoPin
```

Comment:
should we define SPI0_SCK_PIN as EPD_SCK_PIN, same for EPD_SDO_PIN ?

Reply:
Done.